
#include "player_skill_replied_handler.h"

///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE



void SceneSkillClientPlayerRepliedHandler::ReleaseSkill(entt::entity player,const ::ReleaseSkillSkillRequest* request,
	::ReleaseSkillSkillResponse* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE


}



void SceneSkillClientPlayerRepliedHandler::NotifySkillUsed(entt::entity player,const ::SkillUsedS2C* request,
	::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE


}



void SceneSkillClientPlayerRepliedHandler::NotifySkillInterrupted(entt::entity player,const ::SkillInterruptedS2C* request,
	::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE


}



void SceneSkillClientPlayerRepliedHandler::GetSkillList(entt::entity player,const ::GetSkillListRequest* request,
	::GetSkillListResponse* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE


}


